File: URI Online Judg/Beginner/URI-C-Plus-Plus/1010.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){

    int code_1, num_1, code_2, num_2;
    double price_1, price_2, to_be_paid;

    std::cin >> code_1 >> num_1 >> price_1;
    std::cin >> code_2 >> num_2 >> price_2;

    to_be_paid = (num_1 * price_1) + (num_2 * price_2);

    std::cout << "VALOR A PAGAR: R$ " << fixed << setprecision(2) << to_be_paid << '\n';

    return 0;

}

File: URI Online Judg/Beginner/URI-Python/1010.py
code_1, num_1, price_1 = input().split(' ')

code_2, num_2, price_2 = input().split(' ')

code_1, num_1, code_2, num_2 = int(code_1), int(num_1), int(code_2), int(num_2)

price_1, price_2 = float(price_1), float(price_2)

to_be_paid = (num_1 * price_1) + (num_2 * price_2)

print("VALOR A PAGAR: R$ %.2F"%to_be_paid)
